src/main/java/htsjdk/samtools/SAMRecordUtil.java

```diff
@@ -39,7 +39,9 @@
  */
 @Deprecated
```

Comment:
This is good but you also want to use the `@deprecated` Javadoc tag that includes the version number where the API became deprecated. I think that marking a class as deprecated should be sufficient and you don't need to also mark each member. But I guess it couldn't hurt to do that either.

FYI the Java doc on deprecation is here: http://docs.oracle.com/javase/7/docs/technotes/guides/javadoc/deprecation/deprecation.html